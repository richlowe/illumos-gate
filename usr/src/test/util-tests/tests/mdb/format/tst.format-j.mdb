-1=JRjnn
0=JRjnn
1=JRjnn
feedface=JRjnn
badfeedcafe=JRjnn
deadba11e12a=JRjnn
badb100d=JRjnn
baddefec8ed=JRjnn
