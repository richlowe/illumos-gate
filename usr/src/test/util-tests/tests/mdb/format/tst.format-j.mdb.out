                ffffffffffffffff 
                1111111111111111111111111111111111111111111111111111111111111111
                1111111111111111111111111111111111111111111111111111111111111111
                ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
                |||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||+-- bit 0  mask 0x0000000000000001
                ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||+--- bit 1  mask 0x0000000000000002
                |||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||+---- bit 2  mask 0x0000000000000004
                ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||+----- bit 3  mask 0x0000000000000008
                |||||||||||||||||||||||||||||||||||||||||||||||||||||||||||+------ bit 4  mask 0x0000000000000010
                ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||+------- bit 5  mask 0x0000000000000020
                |||||||||||||||||||||||||||||||||||||||||||||||||||||||||+-------- bit 6  mask 0x0000000000000040
                ||||||||||||||||||||||||||||||||||||||||||||||||||||||||+--------- bit 7  mask 0x0000000000000080
                |||||||||||||||||||||||||||||||||||||||||||||||||||||||+---------- bit 8  mask 0x0000000000000100
                ||||||||||||||||||||||||||||||||||||||||||||||||||||||+----------- bit 9  mask 0x0000000000000200
                |||||||||||||||||||||||||||||||||||||||||||||||||||||+------------ bit 10 mask 0x0000000000000400
                ||||||||||||||||||||||||||||||||||||||||||||||||||||+------------- bit 11 mask 0x0000000000000800
                |||||||||||||||||||||||||||||||||||||||||||||||||||+-------------- bit 12 mask 0x0000000000001000
                ||||||||||||||||||||||||||||||||||||||||||||||||||+--------------- bit 13 mask 0x0000000000002000
                |||||||||||||||||||||||||||||||||||||||||||||||||+---------------- bit 14 mask 0x0000000000004000
                ||||||||||||||||||||||||||||||||||||||||||||||||+----------------- bit 15 mask 0x0000000000008000
                |||||||||||||||||||||||||||||||||||||||||||||||+------------------ bit 16 mask 0x0000000000010000
                ||||||||||||||||||||||||||||||||||||||||||||||+------------------- bit 17 mask 0x0000000000020000
                |||||||||||||||||||||||||||||||||||||||||||||+-------------------- bit 18 mask 0x0000000000040000
                ||||||||||||||||||||||||||||||||||||||||||||+--------------------- bit 19 mask 0x0000000000080000
                |||||||||||||||||||||||||||||||||||||||||||+---------------------- bit 20 mask 0x0000000000100000
                ||||||||||||||||||||||||||||||||||||||||||+----------------------- bit 21 mask 0x0000000000200000
                |||||||||||||||||||||||||||||||||||||||||+------------------------ bit 22 mask 0x0000000000400000
                ||||||||||||||||||||||||||||||||||||||||+------------------------- bit 23 mask 0x0000000000800000
                |||||||||||||||||||||||||||||||||||||||+-------------------------- bit 24 mask 0x0000000001000000
                ||||||||||||||||||||||||||||||||||||||+--------------------------- bit 25 mask 0x0000000002000000
                |||||||||||||||||||||||||||||||||||||+---------------------------- bit 26 mask 0x0000000004000000
                ||||||||||||||||||||||||||||||||||||+----------------------------- bit 27 mask 0x0000000008000000
                |||||||||||||||||||||||||||||||||||+------------------------------ bit 28 mask 0x0000000010000000
                ||||||||||||||||||||||||||||||||||+------------------------------- bit 29 mask 0x0000000020000000
                |||||||||||||||||||||||||||||||||+-------------------------------- bit 30 mask 0x0000000040000000
                ||||||||||||||||||||||||||||||||+--------------------------------- bit 31 mask 0x0000000080000000
                |||||||||||||||||||||||||||||||+---------------------------------- bit 32 mask 0x0000000100000000
                ||||||||||||||||||||||||||||||+----------------------------------- bit 33 mask 0x0000000200000000
                |||||||||||||||||||||||||||||+------------------------------------ bit 34 mask 0x0000000400000000
                ||||||||||||||||||||||||||||+------------------------------------- bit 35 mask 0x0000000800000000
                |||||||||||||||||||||||||||+-------------------------------------- bit 36 mask 0x0000001000000000
                ||||||||||||||||||||||||||+--------------------------------------- bit 37 mask 0x0000002000000000
                |||||||||||||||||||||||||+---------------------------------------- bit 38 mask 0x0000004000000000
                ||||||||||||||||||||||||+----------------------------------------- bit 39 mask 0x0000008000000000
                |||||||||||||||||||||||+------------------------------------------ bit 40 mask 0x0000010000000000
                ||||||||||||||||||||||+------------------------------------------- bit 41 mask 0x0000020000000000
                |||||||||||||||||||||+-------------------------------------------- bit 42 mask 0x0000040000000000
                ||||||||||||||||||||+--------------------------------------------- bit 43 mask 0x0000080000000000
                |||||||||||||||||||+---------------------------------------------- bit 44 mask 0x0000100000000000
                ||||||||||||||||||+----------------------------------------------- bit 45 mask 0x0000200000000000
                |||||||||||||||||+------------------------------------------------ bit 46 mask 0x0000400000000000
                ||||||||||||||||+------------------------------------------------- bit 47 mask 0x0000800000000000
                |||||||||||||||+-------------------------------------------------- bit 48 mask 0x0001000000000000
                ||||||||||||||+--------------------------------------------------- bit 49 mask 0x0002000000000000
                |||||||||||||+---------------------------------------------------- bit 50 mask 0x0004000000000000
                ||||||||||||+----------------------------------------------------- bit 51 mask 0x0008000000000000
                |||||||||||+------------------------------------------------------ bit 52 mask 0x0010000000000000
                ||||||||||+------------------------------------------------------- bit 53 mask 0x0020000000000000
                |||||||||+-------------------------------------------------------- bit 54 mask 0x0040000000000000
                ||||||||+--------------------------------------------------------- bit 55 mask 0x0080000000000000
                |||||||+---------------------------------------------------------- bit 56 mask 0x0100000000000000
                ||||||+----------------------------------------------------------- bit 57 mask 0x0200000000000000
                |||||+------------------------------------------------------------ bit 58 mask 0x0400000000000000
                ||||+------------------------------------------------------------- bit 59 mask 0x0800000000000000
                |||+-------------------------------------------------------------- bit 60 mask 0x1000000000000000
                ||+--------------------------------------------------------------- bit 61 mask 0x2000000000000000
                |+---------------------------------------------------------------- bit 62 mask 0x4000000000000000
                +----------------------------------------------------------------- bit 63 mask 0x8000000000000000
                
                
                0               
                0
                0
                
                
                1               
                1
                1
                |
                +-- bit 0 mask 0x1
                
                
                feedface        
                11111110111011011111101011001110
                11111110111011011111101011001110
                ||||||| ||| || |||||| | ||  |||
                ||||||| ||| || |||||| | ||  ||+--- bit 1  mask 0x00000002
                ||||||| ||| || |||||| | ||  |+---- bit 2  mask 0x00000004
                ||||||| ||| || |||||| | ||  +----- bit 3  mask 0x00000008
                ||||||| ||| || |||||| | |+-------- bit 6  mask 0x00000040
                ||||||| ||| || |||||| | +--------- bit 7  mask 0x00000080
                ||||||| ||| || |||||| +----------- bit 9  mask 0x00000200
                ||||||| ||| || |||||+------------- bit 11 mask 0x00000800
                ||||||| ||| || ||||+-------------- bit 12 mask 0x00001000
                ||||||| ||| || |||+--------------- bit 13 mask 0x00002000
                ||||||| ||| || ||+---------------- bit 14 mask 0x00004000
                ||||||| ||| || |+----------------- bit 15 mask 0x00008000
                ||||||| ||| || +------------------ bit 16 mask 0x00010000
                ||||||| ||| |+-------------------- bit 18 mask 0x00040000
                ||||||| ||| +--------------------- bit 19 mask 0x00080000
                ||||||| ||+----------------------- bit 21 mask 0x00200000
                ||||||| |+------------------------ bit 22 mask 0x00400000
                ||||||| +------------------------- bit 23 mask 0x00800000
                ||||||+--------------------------- bit 25 mask 0x02000000
                |||||+---------------------------- bit 26 mask 0x04000000
                ||||+----------------------------- bit 27 mask 0x08000000
                |||+------------------------------ bit 28 mask 0x10000000
                ||+------------------------------- bit 29 mask 0x20000000
                |+-------------------------------- bit 30 mask 0x40000000
                +--------------------------------- bit 31 mask 0x80000000
                
                
                badfeedcafe     
                10111010110111111110111011011100101011111110
                10111010110111111110111011011100101011111110
                | ||| | || |||||||| ||| || |||  | | |||||||
                | ||| | || |||||||| ||| || |||  | | ||||||+--- bit 1  mask 0x00000000002
                | ||| | || |||||||| ||| || |||  | | |||||+---- bit 2  mask 0x00000000004
                | ||| | || |||||||| ||| || |||  | | ||||+----- bit 3  mask 0x00000000008
                | ||| | || |||||||| ||| || |||  | | |||+------ bit 4  mask 0x00000000010
                | ||| | || |||||||| ||| || |||  | | ||+------- bit 5  mask 0x00000000020
                | ||| | || |||||||| ||| || |||  | | |+-------- bit 6  mask 0x00000000040
                | ||| | || |||||||| ||| || |||  | | +--------- bit 7  mask 0x00000000080
                | ||| | || |||||||| ||| || |||  | +----------- bit 9  mask 0x00000000200
                | ||| | || |||||||| ||| || |||  +------------- bit 11 mask 0x00000000800
                | ||| | || |||||||| ||| || ||+---------------- bit 14 mask 0x00000004000
                | ||| | || |||||||| ||| || |+----------------- bit 15 mask 0x00000008000
                | ||| | || |||||||| ||| || +------------------ bit 16 mask 0x00000010000
                | ||| | || |||||||| ||| |+-------------------- bit 18 mask 0x00000040000
                | ||| | || |||||||| ||| +--------------------- bit 19 mask 0x00000080000
                | ||| | || |||||||| ||+----------------------- bit 21 mask 0x00000200000
                | ||| | || |||||||| |+------------------------ bit 22 mask 0x00000400000
                | ||| | || |||||||| +------------------------- bit 23 mask 0x00000800000
                | ||| | || |||||||+--------------------------- bit 25 mask 0x00002000000
                | ||| | || ||||||+---------------------------- bit 26 mask 0x00004000000
                | ||| | || |||||+----------------------------- bit 27 mask 0x00008000000
                | ||| | || ||||+------------------------------ bit 28 mask 0x00010000000
                | ||| | || |||+------------------------------- bit 29 mask 0x00020000000
                | ||| | || ||+-------------------------------- bit 30 mask 0x00040000000
                | ||| | || |+--------------------------------- bit 31 mask 0x00080000000
                | ||| | || +---------------------------------- bit 32 mask 0x00100000000
                | ||| | |+------------------------------------ bit 34 mask 0x00400000000
                | ||| | +------------------------------------- bit 35 mask 0x00800000000
                | ||| +--------------------------------------- bit 37 mask 0x02000000000
                | ||+----------------------------------------- bit 39 mask 0x08000000000
                | |+------------------------------------------ bit 40 mask 0x10000000000
                | +------------------------------------------- bit 41 mask 0x20000000000
                +--------------------------------------------- bit 43 mask 0x80000000000
                
                
                deadba11e12a    
                110111101010110110111010000100011110000100101010
                110111101010110110111010000100011110000100101010
                || |||| | | || || ||| |    |   ||||    |  | | |
                || |||| | | || || ||| |    |   ||||    |  | | +--- bit 1  mask 0x000000000002
                || |||| | | || || ||| |    |   ||||    |  | +----- bit 3  mask 0x000000000008
                || |||| | | || || ||| |    |   ||||    |  +------- bit 5  mask 0x000000000020
                || |||| | | || || ||| |    |   ||||    +---------- bit 8  mask 0x000000000100
                || |||| | | || || ||| |    |   |||+--------------- bit 13 mask 0x000000002000
                || |||| | | || || ||| |    |   ||+---------------- bit 14 mask 0x000000004000
                || |||| | | || || ||| |    |   |+----------------- bit 15 mask 0x000000008000
                || |||| | | || || ||| |    |   +------------------ bit 16 mask 0x000000010000
                || |||| | | || || ||| |    +---------------------- bit 20 mask 0x000000100000
                || |||| | | || || ||| +--------------------------- bit 25 mask 0x000002000000
                || |||| | | || || ||+----------------------------- bit 27 mask 0x000008000000
                || |||| | | || || |+------------------------------ bit 28 mask 0x000010000000
                || |||| | | || || +------------------------------- bit 29 mask 0x000020000000
                || |||| | | || |+--------------------------------- bit 31 mask 0x000080000000
                || |||| | | || +---------------------------------- bit 32 mask 0x000100000000
                || |||| | | |+------------------------------------ bit 34 mask 0x000400000000
                || |||| | | +------------------------------------- bit 35 mask 0x000800000000
                || |||| | +--------------------------------------- bit 37 mask 0x002000000000
                || |||| +----------------------------------------- bit 39 mask 0x008000000000
                || |||+------------------------------------------- bit 41 mask 0x020000000000
                || ||+-------------------------------------------- bit 42 mask 0x040000000000
                || |+--------------------------------------------- bit 43 mask 0x080000000000
                || +---------------------------------------------- bit 44 mask 0x100000000000
                |+------------------------------------------------ bit 46 mask 0x400000000000
                +------------------------------------------------- bit 47 mask 0x800000000000
                
                
                badb100d        
                10111010110110110001000000001101
                10111010110110110001000000001101
                | ||| | || || ||   |        || |
                | ||| | || || ||   |        || +-- bit 0  mask 0x00000001
                | ||| | || || ||   |        |+---- bit 2  mask 0x00000004
                | ||| | || || ||   |        +----- bit 3  mask 0x00000008
                | ||| | || || ||   +-------------- bit 12 mask 0x00001000
                | ||| | || || |+------------------ bit 16 mask 0x00010000
                | ||| | || || +------------------- bit 17 mask 0x00020000
                | ||| | || |+--------------------- bit 19 mask 0x00080000
                | ||| | || +---------------------- bit 20 mask 0x00100000
                | ||| | |+------------------------ bit 22 mask 0x00400000
                | ||| | +------------------------- bit 23 mask 0x00800000
                | ||| +--------------------------- bit 25 mask 0x02000000
                | ||+----------------------------- bit 27 mask 0x08000000
                | |+------------------------------ bit 28 mask 0x10000000
                | +------------------------------- bit 29 mask 0x20000000
                +--------------------------------- bit 31 mask 0x80000000
                
                
                baddefec8ed     
                10111010110111011110111111101100100011101101
                10111010110111011110111111101100100011101101
                | ||| | || ||| |||| ||||||| ||  |   ||| || |
                | ||| | || ||| |||| ||||||| ||  |   ||| || +-- bit 0  mask 0x00000000001
                | ||| | || ||| |||| ||||||| ||  |   ||| |+---- bit 2  mask 0x00000000004
                | ||| | || ||| |||| ||||||| ||  |   ||| +----- bit 3  mask 0x00000000008
                | ||| | || ||| |||| ||||||| ||  |   ||+------- bit 5  mask 0x00000000020
                | ||| | || ||| |||| ||||||| ||  |   |+-------- bit 6  mask 0x00000000040
                | ||| | || ||| |||| ||||||| ||  |   +--------- bit 7  mask 0x00000000080
                | ||| | || ||| |||| ||||||| ||  +------------- bit 11 mask 0x00000000800
                | ||| | || ||| |||| ||||||| |+---------------- bit 14 mask 0x00000004000
                | ||| | || ||| |||| ||||||| +----------------- bit 15 mask 0x00000008000
                | ||| | || ||| |||| ||||||+------------------- bit 17 mask 0x00000020000
                | ||| | || ||| |||| |||||+-------------------- bit 18 mask 0x00000040000
                | ||| | || ||| |||| ||||+--------------------- bit 19 mask 0x00000080000
                | ||| | || ||| |||| |||+---------------------- bit 20 mask 0x00000100000
                | ||| | || ||| |||| ||+----------------------- bit 21 mask 0x00000200000
                | ||| | || ||| |||| |+------------------------ bit 22 mask 0x00000400000
                | ||| | || ||| |||| +------------------------- bit 23 mask 0x00000800000
                | ||| | || ||| |||+--------------------------- bit 25 mask 0x00002000000
                | ||| | || ||| ||+---------------------------- bit 26 mask 0x00004000000
                | ||| | || ||| |+----------------------------- bit 27 mask 0x00008000000
                | ||| | || ||| +------------------------------ bit 28 mask 0x00010000000
                | ||| | || ||+-------------------------------- bit 30 mask 0x00040000000
                | ||| | || |+--------------------------------- bit 31 mask 0x00080000000
                | ||| | || +---------------------------------- bit 32 mask 0x00100000000
                | ||| | |+------------------------------------ bit 34 mask 0x00400000000
                | ||| | +------------------------------------- bit 35 mask 0x00800000000
                | ||| +--------------------------------------- bit 37 mask 0x02000000000
                | ||+----------------------------------------- bit 39 mask 0x08000000000
                | |+------------------------------------------ bit 40 mask 0x10000000000
                | +------------------------------------------- bit 41 mask 0x20000000000
                +--------------------------------------------- bit 43 mask 0x80000000000
                
                
